# Da Física Clássica à Teoria Quântica

Acreditando que átomos eram bolas elásticas, conseguiram explicar vários fenômenos observados. Mas e a estabilidade das moléculas? Como se ligam e continuam assim?
Max Plank por volta de 1900 descobriu que a energia era emitida em pacotes, ou seja, em determinadas quantidades. Uma analogia feita pelo físico Marcelo Gleiser que acho bem legal é a seguinte:\
O motivo de demorarmos tanto a perceber que a energia vem sempre em pacotes é porque vivemos em um mundo de milionários, sem contatos com moedas de 1 centavo. Mas 1 centavo existe e todo o nosso dinheiro é um múltiplo dele.\

## Ondas
Uma onda é uma forma de transmitir energia, vibracionalmente. Ela possui um comprimento, frequência e amplitude.

<img src="onda.png" width=50% >

A sua velocidade pode ser expressa como:

\begin{equation}
v = \lambda f
\end{equation}

Em que $\lambda$ é o comprimento de onda e $f$ a frequência (em Hz).
Podemos também escrever da seguinte forma:

\begin{equation}
f = \frac{1}{T}
\end{equation}

\begin{equation}
v = \frac{\lambda}{T}
\end{equation}

Em que T é o período.

In [ ]:
#v = lam * f
#lam = v/f
#f = v/lam

In [1]:
#Definindo as funções equivalentes (isolando v, lambda ou f):

def velocidade (lam, f):
    return lam * f

def comprimento (v, f):
    return v/f

def frequencia (v, lam):
    return v/lam


Exemplo 1 - O comprimento de luz verde tem comprimento próximo a 522nm. Qual o valor de f?\

Velocidade da luz: 3 x 10⁸ m/s.

In [7]:
v = 3e8
lam = 522e-9 #1 nano metro = 10^-9 m
print('Frequência = ', frequencia (v, lam))

Frequência =  574712643678160.9


Ou seja, passam 574712643678160.9 "ondas" a cada segundo por um mesmo ponto.\

Exemplo 2 - Qual o comprimento de onda de uma onda eletromagnética com $f = 3.64 x 10^7$ Hz?

In [8]:
f = 3.64e7

print('Comprimento = ', comprimento(v, f))

Comprimento =  8.241758241758241


# Teoria Quântica de Planck

Planck chamou de quantun a menor quantidade de energia que pode ser emitida ou absorvida na forma de radiação eletromagnética. Essa energia pode ser expressa como:

\begin{equation}
E = hf
\end{equation}

Ou

\begin{equation}
E = h\frac{c}{\lambda}
\end{equation}

Em que $f$ é a frequência, $h$ a constante de Planck, $c$ a velocidade (da luz) e $\lambda$ o comprimento de onda. Sendo que $h = 6.63 x 10^{-34}$ Jxs.

In [10]:
def Energia_f(f):
    
    '''A frequência deve ser em Hz'''
    
    return 6.63e-34 * f

def Energia_l(lam):
    
    '''O comprimento deve ser em metro'''
    
    return 6.63e-34 * (3e8/lam)

Exemplo 1 - Qual a energia de um fóton com $\lambda = 5.00 x 10^{4}$ nm (infravermelho) e outro com $\lambda = 5.00 x 10^{-2}$ nm (raio X).

In [18]:
#Como foi dado o comprimento, usaremos Energia_l

print('Energia do fóton 1: ',Energia_l(5e-5),'j') #lembrando que foi dado em nm e não em m. Aqui fizemos a 
print('Energia do fóton 2: ',Energia_l(5e-11),'j') #conversão

Energia do fóton 1:  3.978e-21 j
Energia do fóton 2:  3.978e-15 j


Exemplo 2 - A energia de um fóton é de $5.87 x 10^{-20} j$. Qual o seu comprimento de onda?

In [20]:
#Fazendo uma nova função
def Comprimento(E):
    return 6.63e-34*3e8/E

In [25]:
print('Comprimento de onda: ', Comprimento(5.87e-20)/1e-9, 'nm.')
print('Comprimento de onda (em m): ', Comprimento(5.87e-20), 'm')

Comprimento de onda:  3388.4156729131173 nm.
Comprimento de onda (em m):  3.3884156729131173e-06 m


# Efeito Fotoelétrico

Em 1905, Albert Einstein propôs uma solução para o Efeito Fotoelétrico (eletrons são expelidos da superfície de alguns metais quando expostos a uma luz a partir de uma frequência limite).
Ele sugeriu que um feixe de luz seria um feixe de partículas, os fótons.
Os elétrons estão aprisionados em um metal e para arrancá-los é necessário uma certa frequência suficientemente elevada. Se a frequência dos fótons for tal que $hf$ é exatamente igual à energia para arrancá-los, eles serão apenas arrancados. Se $hf$ for maior que essa energia necessária, eles serão arrancados e vão adquirir energia cinética.
Podemos expressar como:

\begin{equation}
hf = K + U
\end{equation}

Em que U é a energia cinética e K a energia para arrancá-lo, lembrando que $hf$ é a energia que vimos acima. Rearranjando:

\begin{equation}
U = hf - K
\end{equation}

Ao escrever a equação dessa forma, podemos entender melhor. Quanto mais energético o fóton (maior for  a sua frequência), maior será a energia do elétron expelido.

O problema disso é que consideramos a luz agora não como uma onda, mas sim como uma partícula! Como essa explicação é condizente com o que se observa nos experimentos, admite-se que a luz tem as propriedas de onda e partícula! É essa a dualidade tão comentada.

In [13]:
#Energia cinética do elétron

def U(f, K):
    ''' f em Hz e K em joule'''
    return 6.63e-34 * f - K

Fonte do texto:\
CHANG, Raymond. "Química Geral, conceitos essenciais". 4 ed. Porto Alegre: AMGH, 2010.